test/unit_CartesianProduct.jl

```diff
@@ -387,6 +387,10 @@ for N in [Float64, Float32]
     cp = Interval(N(0), N(2)) × Hyperrectangle(N[2, 3], N[1, 1])
     @test project(cp, 1:2) == Hyperrectangle(N[1, 2], N[1, 1])
     @test project(cp, 2:3) == Hyperrectangle(N[2, 3], N[1, 1])
+
+    cp = Interval(N(0), N(2)) × Zonotope(N[2, 3], N[1 0; 0 1])
+    @test isequivalent(project(cp, 1:2), Zonotope(N[1, 2], N[1 0; 0 1]))
+    @test isequivalent(project(cp, 2:3), Zonotope(N[2, 3], N[1 0; 0 1]))
```

Comment:
Just to understand: is the `isequivalent` needed? From the code I would expect that you have equality here.

Reply:
There are cases where the zonotope that the function gives has sparce matrices, so to have a cleaner code I used `isequivalent`

Reply:
I see, thanks!